2024-12-27 14:53:46,872 - VideoFlow - INFO - logger.py:68 - Logger initialisé
2024-12-27 14:53:46,872 - VideoFlow.Main - INFO - main.py:15 - Démarrage de l'application
2024-12-27 14:53:46,973 - VideoFlow.MainWindow - INFO - main_window.py:11 - Initialisation de la fenêtre principale
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - INFO - plugin_manager.py:18 - PluginManager initialisé avec le dossier: /Users/nico/Documents/videoFlow/src/plugins
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:19 - Le dossier existe: True
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:22 - Contenu du dossier plugins:
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:31 -   - Fichier: __init__.py
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:26 -   - Dossier: tree_manager
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:27 -     Contenu:
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:26 -   - Dossier: duplicate_finder
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:27 -     Contenu:
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:29 -       - window.py
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:29 -       - video_hasher.py
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:29 -       - __init__.py
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:29 -       - plugin.py
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:29 -       - data_manager.py
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:26 -   - Dossier: video_merger
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:27 -     Contenu:
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:26 -   - Dossier: regex_renamer
2024-12-27 14:53:46,973 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:27 -     Contenu:
2024-12-27 14:53:46,974 - VideoFlow.MainWindow - DEBUG - main_window.py:16 - Initialisation de l'interface utilisateur
2024-12-27 14:53:46,974 - VideoFlow.MainWindow - DEBUG - main_window.py:27 - Chargement des plugins
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - INFO - plugin_manager.py:37 - Début du chargement des plugins
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:47 - Dossiers de plugins trouvés: ['tree_manager', 'duplicate_finder', 'video_merger', 'regex_renamer']
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:54 - Vérification du dossier tree_manager:
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:55 -   - plugin.py existe: False
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:56 -   - __init__.py existe: False
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - WARNING - plugin_manager.py:66 - Fichiers manquants dans: /Users/nico/Documents/videoFlow/src/plugins/tree_manager
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - WARNING - plugin_manager.py:68 -   - plugin.py manquant
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - WARNING - plugin_manager.py:70 -   - __init__.py manquant
2024-12-27 14:53:46,974 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:54 - Vérification du dossier duplicate_finder:
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:55 -   - plugin.py existe: True
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:56 -   - __init__.py existe: True
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:59 - Tentative de chargement du plugin depuis: /Users/nico/Documents/videoFlow/src/plugins/duplicate_finder/plugin.py
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:87 - Chargement du plugin depuis: /Users/nico/Documents/videoFlow/src/plugins/duplicate_finder/plugin.py
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:99 - Module chargé. Contenu:
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - Logger: <class 'type'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - Plugin: <class 'abc.ABCMeta'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - PluginInterface: <class 'abc.ABCMeta'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - QColor: <class 'PyQt6.sip.wrappertype'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - QIcon: <class 'PyQt6.sip.wrappertype'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - QPushButton: <class 'PyQt6.sip.wrappertype'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - QSize: <class 'PyQt6.sip.wrappertype'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - Qt: <class 'PyQt6.sip.wrappertype'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - __builtins__: <class 'dict'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - __cached__: <class 'str'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - __doc__: <class 'NoneType'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - __file__: <class 'str'>
2024-12-27 14:53:46,975 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - __loader__: <class '_frozen_importlib_external.SourceFileLoader'>
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - __name__: <class 'str'>
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - __package__: <class 'str'>
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - __spec__: <class '_frozen_importlib.ModuleSpec'>
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - logger: <class 'logging.Logger'>
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:101 -   - os: <class 'module'>
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:109 - Classe Plugin valide trouvée dans /Users/nico/Documents/videoFlow/src/plugins/duplicate_finder/plugin.py
2024-12-27 14:53:46,976 - VideoFlow.DuplicateFinder.Plugin - DEBUG - plugin.py:12 - Initialisation du plugin DuplicateFinder
2024-12-27 14:53:46,976 - VideoFlow.DuplicateFinder.Plugin - DEBUG - plugin.py:17 - get_name() appelé
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - INFO - plugin_manager.py:113 - Plugin chargé avec succès: Duplicate Finder
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:54 - Vérification du dossier video_merger:
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:55 -   - plugin.py existe: False
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:56 -   - __init__.py existe: False
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - WARNING - plugin_manager.py:66 - Fichiers manquants dans: /Users/nico/Documents/videoFlow/src/plugins/video_merger
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - WARNING - plugin_manager.py:68 -   - plugin.py manquant
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - WARNING - plugin_manager.py:70 -   - __init__.py manquant
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:54 - Vérification du dossier regex_renamer:
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:55 -   - plugin.py existe: False
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:56 -   - __init__.py existe: False
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - WARNING - plugin_manager.py:66 - Fichiers manquants dans: /Users/nico/Documents/videoFlow/src/plugins/regex_renamer
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - WARNING - plugin_manager.py:68 -   - plugin.py manquant
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - WARNING - plugin_manager.py:70 -   - __init__.py manquant
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - INFO - plugin_manager.py:72 - Plugins chargés: ['Plugin']
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:76 - Plugin Plugin:
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:78 -   - Méthode: __init__
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:78 -   - Méthode: get_button
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:78 -   - Méthode: get_name
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:78 -   - Méthode: show_window
2024-12-27 14:53:46,976 - VideoFlow.PluginManager - DEBUG - plugin_manager.py:130 - Retourne 1 plugins
2024-12-27 14:53:46,976 - VideoFlow.MainWindow - INFO - main_window.py:30 - Nombre de plugins chargés: 1
2024-12-27 14:53:46,976 - VideoFlow.DuplicateFinder.Plugin - DEBUG - plugin.py:17 - get_name() appelé
2024-12-27 14:53:46,976 - VideoFlow.MainWindow - DEBUG - main_window.py:39 - Ajout du plugin Duplicate Finder à la position (0, 0)
2024-12-27 14:53:46,976 - VideoFlow.DuplicateFinder.Plugin - DEBUG - plugin.py:21 - Création du bouton DuplicateFinder
2024-12-27 14:53:46,976 - VideoFlow.DuplicateFinder.Plugin - DEBUG - plugin.py:27 - Chemin de l'icône: /Users/nico/Documents/videoFlow/resources/icons/duplicate_finder.svg
2024-12-27 14:53:46,980 - VideoFlow.DuplicateFinder.Plugin - DEBUG - plugin.py:33 - Icône ajoutée au bouton
2024-12-27 14:53:46,980 - VideoFlow.DuplicateFinder.Plugin - DEBUG - plugin.py:62 - Bouton DuplicateFinder créé et configuré
2024-12-27 14:53:46,995 - VideoFlow.MainWindow - DEBUG - main_window.py:55 - Dimensions de la fenêtre: 450x140
2024-12-27 14:53:46,995 - VideoFlow.MainWindow - INFO - main_window.py:59 - Interface utilisateur initialisée
2024-12-27 14:53:47,103 - VideoFlow.Main - INFO - main.py:20 - Fenêtre principale affichée
2024-12-27 14:53:47,919 - VideoFlow.DuplicateFinder.Plugin - DEBUG - plugin.py:66 - Ouverture de la fenêtre DuplicateFinder
2024-12-27 14:54:03,741 - VideoFlow.DuplicateFinder.DataManager - INFO - data_manager.py:89 - Données effacées avec succès
2024-12-27 14:54:03,742 - VideoFlow.DuplicateFinder.Window - INFO - window.py:500 - Toutes les empreintes ont été effacées
2024-12-27 14:55:29,953 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,954 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,955 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,955 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,955 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,955 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,955 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,955 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,955 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,955 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,956 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,956 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:55:29,956 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,786 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,787 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,787 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,788 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,788 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,788 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,789 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,790 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,793 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,794 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,797 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,798 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,799 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,799 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,800 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,801 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,802 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,802 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,803 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,803 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,804 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
2024-12-27 14:59:59,804 - VideoFlow.DuplicateFinder.DataManager - ERROR - data_manager.py:49 - Erreur lors de la sauvegarde des données : Object of type ndarray is not JSON serializable
